styles/style.css

```diff
@@ -0,0 +1,412 @@
+    height:1200px; 
+    padding-bottom: 2%; 
+    overflow:hidden; 
+    position:relative;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.fling-minislid img{ 
+    position:absolute; 
+    animation:fling-minislide 30s infinite; 
+    opacity:0; 
+    width: 100%; 
+    height: auto;
+}
+
+
+h2, .page-subtitle{
+max-width: 100 rem;
+margin: 0 auto;
+padding: 0 1.25rem;
+font-family: 'Lato', sans-serif;
+text-align: center;
+color:rgb(219, 68, 196);
+}
+
+.images{
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+@keyframes fling-minislid {25%{opacity:1;} 40%{opacity:0;}}
+.fling-minislid img:nth-child(4){animation-delay:0s;}
+.fling-minislid img:nth-child(3){animation-delay:10s;}
+.fling-minislid img:nth-child(2){animation-delay:20s;}
+.fling-minislid img:nth-child(1){animation-delay:30s;}
+
+
+@keyframes tonext {
+    75% {
+    left: 0;
+    }
+    95% {
+    left: 100%;
+    }
+    98% {
+    left: 100%;
+    }
+    99% {
+    left: 0;
+    }
+}
+
+@keyframes tostart {
+    75% {
+    left: 0;
+    }
+    95% {
+    left: -300%;
+    }
+    98% {
+    left: -300%;
+    }
+    99% {
+    left: 0;
+    }
+}
+
+@keyframes snap {
+    96% {
+    scroll-snap-align: center;
+    }
+    97% {
+    scroll-snap-align: none;
+    }
+    99% {
+    scroll-snap-align: none;
+    }
+    100% {
+    scroll-snap-align: center;
+    }
+}
+
+.logo{
+    position:left;
+}
+
+.navbar {
+    background-color: rgb(170, 91, 212);
+    overflow: hidden;
+    position:static;
+    color:rgba(255, 255, 255, 0.83);
+    text-align: center;
+}
+
+.navbar a {
+    float:left;
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+}
+
+.navbar a:hover {
+    background-color: rgb(133, 220, 233);
+    color: rgb(188, 50, 238);
+}
+
+.navbar a.active {
+    background-color: #7b07ad;
+    color: white;
+}
+
+.carousel {
+    box-sizing: border-box;
+    scrollbar-color: transparent transparent;
+    scrollbar-width: 0px;
+}
+
+.carousel::-webkit-scrollbar {
+    width: 0;
+}
+
+.carousel::-webkit-scrollbar-track {
+    background: transparent;
+}
+
+.carousel::-webkit-scrollbar-thumb {
+    background: transparent;
+    border: none;
+}
+
+.carousel {
+    -ms-overflow-style: none;
+}
+
+ol, li {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+
+.carousel {
+    position: relative;
+    padding-top: 75%;
+    filter: drop-shadow(0 0 10px #0003);
+    perspective: 100px;
+}
+
+.carousel__viewport {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    display: flex;
+    overflow-x: scroll;
+    counter-reset: item;
+    scroll-behavior: smooth;
+    scroll-snap-type: x mandatory;
+}
+
+.carousel__slide {
+    position: relative;
+    flex: 0 0 100%;
+    width: 100%;
+    background-color: rgb(180, 153, 213);
+    counter-increment: item;
+}
+
+.carousel__slide:nth-child(1) {
```

Comment:
There's a lot of repeated code for the carousel in here. I would recommend finding ways to DRY your code a bit,